aten/src/ATen/native/native_functions.yaml

```diff
@@ -244,14 +244,15 @@
     CPU: native_dropout_cpu
     CUDA: native_dropout_cuda
     NestedTensorCPU, NestedTensorCUDA: native_dropout_nested
-  tags: nondeterministic_seeded, canonical
+  tags: [nondeterministic_seeded, canonical]
```

Comment:
Hmm. This is OK but I am fundamentally mistrustful of leaning too much on YAML's built in parser; splitting on the comma was not too hard to do either haha

Reply:
Happy to do either but this seemed more in precedent of how we split everything else and has better highlighting in a code editor. Btw I had a small script to edit yaml files that was helpful here I can share.